cmd/run.go

```diff
@@ -160,6 +153,21 @@ var RunCommand = cli.Command{
 	},
 }
 
+func loadEnvVars(envName string, heritageName string) (map[string]string, error) {
+	result := make(map[string]string)
+	if len(envName) > 0 {
+		env, err := LoadEnvironment(envName)
+		if err != nil {
+			return nil, err
+		}
+		heritageName = env.Name
```

Comment:
I'm no Go expert, but this looks to me like we're writing a value to the `heritageName` argument, and never reading it inside of this function. But Go has no pass-by-reference, so the assignment does nothing, right?

Reply:
You are right. It is a totally useless line. I'll get rid of it